src/docs/content/docs/_index.md

```diff
@@ -4,5 +4,25 @@ show_menu = true
 
 ## Welcome to the Cassandra Reaper documentation!
 
+### Overview
 
+Reaper is an open source tool that aims to schedule and orchestrate repairs of Apache Cassandra clusters. It improves on a repair process based on the classic nodetool repair by allowing
 
+* Split repair jobs into a tunable number of segments.
+* Back-pressure handling through monitoring running repairs and pending compactions.
+* Pause or cancel repairs and tracks progress precisely.
+
+Reaper ships with a REST API, a command line tool and a web UI.
+
+The document includes instructions on how build, install, and configure Reaper correctly.
```

Comment:
"The document" ?

This document, or This documentation…

Reply:
+1 to what @michaelsembwever said

Reply:
Agreed.